test/support/utils.h

```diff
@@ -792,14 +795,16 @@ transform_reduce_serial(InputIterator first, InputIterator last, T init, BinaryO
     return init;
 }
 
-static const char*
-done()
+int
+done(bool is_passed = true)
```

Comment:
a caller passes an integer (macro value)..  :"done" is accepting "bool"...  please, arrange types

Reply:
Done